Update readme (#144)
equinor/xun: README.md

# Xùn

```
█████████████

█████████████

█████   █████

https://en.wikipedia.org/wiki/Bagua
```

[![equinor](https://circleci.com/gh/equinor/xun.svg?style=shield)](https://app.circleci.com/pipelines/github/equinor/xun)

> Distributed functional python framework

Author: ```suggestion
Xun is a distributed and functional Python framework for cluster compute. Rather than focusing on batching jobs, xun is about defining values declaratively.
```

    pass

@xun.function()
def xun_function():
    not_installed()            # Not OK
    not_installed_but_shared() # OK
    radians(180)               # OK because the function is builtin
    np.array([1, 2, 3])        # OK because the function is defined in an installed module
```

Because xun functions are pickled, any function they reference must either be installed on the system or be represented differently. `xun` comes with a decorator, `@xun.make_shared`, that can make many functions serializable.

## Function Scope and Best Practices

>  * You should only reference things outside the function that would not change the outcome of the call if changed. Global scope is not considered when identifying results, thus changes to the global scope might give undesired results.

Author: ```suggestion
 * You should only reference global scope from a function that would not change the outcome of the function if changed. Global scope is not considered when identifying results, thus changes to the global scope might give undesired results.
```
